lib/timed_semaphore.rb

```diff
@@ -8,17 +8,17 @@
 # @author ssuljic
 #
 # @example
-#   threads = []
-#   semaphore = TimedSemaphore.new(2, 3)
+# threads = []
```

Comment:
Don't worry, there is a first time for everything :) I opened a new pull request containing your changes which I'm going to merge. Thanks for your contribution! :)